graphql/resolvers.go

```diff
@@ -712,6 +712,17 @@ func (r *queryResolver) PatchBuildVariants(ctx context.Context, patchID string)
 	return result, nil
 }
 
+func (r *queryResolver) CommitQueue(ctx context.Context, id string) (*restModel.APICommitQueue, error) {
+	commitQueue, err := r.sc.FindCommitQueueByID(id)
+	if err != nil {
+		if err.Error() == fmt.Sprintf("no commit queue found for '%s'", id) {
+			return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("Error finding commit Queue for `%s`: %s", id, error.Error(err)))
+		}
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error finding commit Queue for `%s`: %s", id, error.Error(err)))
```

Comment:
After checking out the graphql config it makes sense to return 2 separate error codes here. Since Apollo Client will automatically retry failed queries that return a  >= 500 error status code but wont for anything else